renderer/index.js

```diff
@@ -251,6 +251,35 @@ function addTorrentEvents (torrent) {
   }
 }
 
+function restoreSession () {
+  cfg.read(function (err, data) {
+    if (err) console.error(err)
+    state.saved = data
+    if (!state.saved.torrents) state.saved.torrents = []
+    state.saved.torrents.forEach(function (torrent) {
+      dispatch('addTorrent', torrent.magnetURI)
+    })
+    update()
+  })
+}
+
+function saveTorrents () {
+  state.saved.torrents = state.client.torrents.map(function (torrent) {
+    return {
+      name: torrent.name,
+      magnetURI: torrent.magnetURI,
+      path: torrent.path
+    }
+  })
+
+  cfg.write({
+    torrents: state.saved.torrents
+  }, function (err) {
```

Comment:
Thought that might lead to overwrites in the future, but that would be fine for now.